config.go

```diff
@@ -87,16 +86,35 @@ type Settings struct {
 func (c *Settings) SectionId() string { return ConfigDocID }
 
 func (c *Settings) Get() error {
-	err := db.FindOneQ(ConfigCollection, db.Query(byId(c.SectionId())), c)
-	if err != nil && err.Error() == errNotFound {
-		*c = Settings{}
-		return nil
+	env := GetEnvironment()
+	ctx, cancel := env.Context()
+	defer cancel()
+	coll := env.DB().Collection(ConfigCollection)
+
+	res := coll.FindOne(ctx, byId(c.SectionId()))
```

Comment:
Not sure why but when I run this locally this returns no docs found, even though I'm using the same config/db that I've always used

Reply:
are you loading from a settings override file? what's the mongodb uri that you're using?

Reply:
No file or URI. It should default to localhost:27017 right? That's the port my mongod is running on